Readme.md

```diff
@@ -4,6 +4,10 @@
 
 ## Changelog
 
+### [14.1.2] - 2020-03-27
+
+* Aligned error codes with current version of the API webservice.
+
 ### [14.1.1] - 2020-03-09
 
 * Tags are now always returned when applying events.
```

Comment:
Remember to add a footnote line in the bottom with a link to the diff:
```
[14.1.2]: https://github.com/deondigital/api-client/compare/v14.1.1...v14.1.2

Reply:
Thanks